apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -50,6 +50,12 @@
       </includes>
     </fileSet>
     <fileSet filtered="true" encoding="UTF-8">
+      <directory>src/main/resources</directory>
+      <includes>
+        <include>**/apexlog4j.properties</include>
```

Comment:
This log4j.properties should only be used in the context of the running of this particular apa. If not, something is seriously wrong. I do not recommend using a custom log4j properties filename.